scripts/tools/nrfconnect/nrfconnect_factory_data.schema

```diff
@@ -5,14 +5,17 @@
     "type": "object",
     "required": [
         "sn",
+        "vid",
+        "pid",
+        "v_name",
```

Comment:
Can we change it to full vendor_name and product_name? p_name is somewhat unclear.